portfolio/src/main/webapp/map.js

```diff
@@ -12,8 +12,11 @@ function createMap() {
       title: "Stan the T-Rex"
     });
 
+    const trexInfoWindow = new google.maps.InfoWindow({content: 'This is Stan, the T-Rex statue.'});
+
     trexMarker.addListener("click", function() {
         toggleBounce(trexMarker);
+        trexInfoWindow.open(map, trexMarker);
```

Comment:
If you click again would it close?

Reply:
![image](https://user-images.githubusercontent.com/49228945/84614420-a1544b80-aef8-11ea-8c3c-6aaf10aa41f6.png)
There is a close tab on the marker so that I can close it via clicking on the close tab